pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -467,17 +465,19 @@ func (a *ofFlowAction) Meter(meterID uint32) FlowBuilder {
 }
 
 // Learn is an action which adds or modifies a flow in an OpenFlow table.
-func (a *ofFlowAction) Learn(id uint8, priority uint16, idleTimeout, hardTimeout uint16, cookieID uint64) LearnAction {
+func (a *ofFlowAction) Learn(id uint8, priority uint16, idleTimeout, hardTimeout uint16, cookieID uint64, isIPv6 bool) LearnAction {
```

Comment:
Before this change, it's possible to have a flow that works for both v4 and v6. Now it requires ip family to be specified, even I don't use them in the learned flow. Does it make sense?

Reply:
You are right, some actions can work for both v4 and v6. I will change this back.

Reply:
Done